src/shared/loggerUtils.ts

```diff
@@ -11,39 +11,38 @@ import * as filesystemUtilities from './filesystemUtilities'
 import * as l from './logger'
 
 export class TestLogger {
-    private readonly _logger: l.Logger
-    private readonly _logfile: string
-
     // initializes a default logger. This persists through all tests.
     // initializing a default logger means that any tested files with logger statements will work.
     // as a best practice, please initialize a TestLogger before running tests on a file with logger statements.
-    private constructor(logfile: string, logger: l.Logger) {
-        this._logger = logger
-        this._logfile = logfile
-    }
+    private constructor(
+        private readonly logFolder: string,
+        private readonly logfilePath: string,
+        private readonly logger: l.Logger
+    ) { }
 
     // cleanupLogger clears out the logger's transports, but the logger will still exist as a default
     // this means that the default logger will still work for other files but will output an error
     public async cleanupLogger(): Promise<void> {
-        this._logger.releaseLogger()
-        if (await filesystemUtilities.fileExists(this._logfile)) {
-            await del(this._logfile, { force: true })
+        this.logger.releaseLogger()
+        if (await filesystemUtilities.fileExists(this.logFolder)) {
+            await del(this.logFolder, { force: true })
         }
     }
 
     public async logContainsText(str: string): Promise<boolean> {
-        const logText = await filesystemUtilities.readFileAsString(this._logfile as string)
+        const logText = await filesystemUtilities.readFileAsString(this.logfilePath)
 
-        return(logText.includes(str))
+        return (logText.includes(str))
```

Comment:
Was about to ask why there were parenthesis around the return statement and then I realized that I was the one that put those in :/

Mind fixing that for me?